js-example/index.html

```diff
@@ -23,4 +23,12 @@
   <body></body>
   <script src="../dist/index.js"></script>
   <script src="output/index.js"></script>
+  <script>
+    setTimeout(function() {
+      const iv = new Uint8Array(4);
+      window.crypto.getRandomValues(iv);
+      const result = CardanoCrypto.PaperWallet.scramble(iv, '', "legal winner thank year wave sausage worth useful legal winner thank yellow");
+      console.log(result);
+    }, 1000)
+  </script>
```

Comment:
The expected result here is:

`abandon abandon abandon win nest chef want salt join shove minor december miss oak name`

can this bit be moved to a test suite in `js` directory ? Might be worth adding unit tests with all the test vectors.

Reply:
Yes, moving this into a js test suite is another goal 😉